plugins/UM3NetworkPrinting/resources/qml/PrintJobContextMenu.qml

```diff
@@ -189,7 +196,7 @@ Item {
         icon: StandardIcon.Warning;
         onYes: printJob.setState("abort");
         standardButtons: StandardButton.Yes | StandardButton.No;
-        text: printJob ? catalog.i18nc("@label %1 is the name of a print job.", "Are you sure you want to abort %1?").arg(printJob.name) : "";
+        text: printJob && printJob.name ? catalog.i18nc("@label %1 is the name of a print job.", "Are you sure you want to abort %1?").arg(printJob.name) : "";
```

Comment:
Why does it matter if the name of the printjob is empty? Shouldn't that be handled in the python part instead of the frontend?

Reply:
Because QML will complain if printJob is null that it has no name property. You'd think that simply checking for printJob would be enough because it evaluates as null but apparently not...

Reply:
So, in that case, shouldn't the check be printJob != null ? catalog.i18nc("@label %1 is the name of a print job.", "Are you sure you want to abort %1?").arg(printJob.name) : ""

Reply:
You would think, yes, but the reason specifically I added it is to silence the fact that without it, if print job is null, Qt still warns "Cannot read property 'name' of [null]" or whatever.

Unless the point is that Qt evaluates a variable defined as `null` to be truthy...?